src/alire/alire-properties-from_toml.ads

```diff
@@ -79,6 +79,49 @@ package Alire.Properties.From_TOML is
                     Website => True,
                     others  => False);
 
+   --  Cardinalities say if a property must be a single value or an array.
+   --  Since we store properties always as elements in a list, we lose this
+   --  information after load so we use this not only during loading to enforce
+   --  index correctness, but also during exporting to ensure the proper type
+   --  (atom/array) is created.
+
+   type Cardinalities is
+     (Unique,
+      --  Must be a single value
+      Multiple_Deprecated,
```

Comment:
The licenses array form has been deprecated since Alire 1.0, maybe it could be removed in Alire 3.0?

Reply:
Yes, and surely we shouldn't have any incompatible release in the index at this point